integration/spark/src/main/scala/org/apache/carbondata/spark/util/GlobalDictionaryUtil.scala

```diff
@@ -629,22 +631,32 @@ object GlobalDictionaryUtil extends Logging {
     // filepath regex, look like "/path/*.dictionary"
     if (filePath.getName.startsWith("*")) {
       val dictExt = filePath.getName.substring(1)
-      val listFiles = filePath.getParentFile.listFiles()
-      if (listFiles.exists(file =>
-        file.getName.endsWith(dictExt) && file.getSize > 0)) {
-        true
+      if (filePath.getParentFile.exists()) {
+        val listFiles = filePath.getParentFile.listFiles()
+        if (listFiles.exists(file =>
+          file.getName.endsWith(dictExt) && file.getSize > 0)) {
+          true
+        } else {
+          logWarning("[ALL_DICTIONARY] No dictionary files found or empty dictionary files! " +
+            "Won't generate new dictionary.")
+          false
+        }
       } else {
-        logInfo("No dictionary files found or empty dictionary files! " +
-          "Won't generate new dictionary.")
-        false
+        throw new FileNotFoundException(
+          "[ALL_DICTIONARY] The given dictionary file path not found!")
       }
     } else {
-      if (filePath.exists() && filePath.getSize > 0) {
-        true
+      if (filePath.exists()) {
+        if (filePath.getSize > 0) {
+          true
+        } else {
+          logWarning("[ALL_DICTIONARY] No dictionary files found or empty dictionary files! " +
+            "Won't generate new dictionary.")
+          false
+        }
       } else {
-        logInfo("No dictionary files found or empty dictionary files! " +
-          "Won't generate new dictionary.")
-        false
+        throw new FileNotFoundException(
+          "[ALL_DICTIONARY] The given dictionary file path not found!")
```

Comment:
Correct english grammar in the log messages.
The given dictionary file path not found! =>The given dictionary file path **is** not found!